pkgs/build-support/rust/test/import-cargo-lock/git-dependency-rev-with-hash/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, rustPlatform }:
+let
+  fs = lib.fileset;
```

Comment:
I think `fileset` has been temporarily forbidden from being used in nixpkgs, so you probably have to use something else. (e.g. putting all required files and directories into a new base directory, and using that as the `src` for the derivation

Reply:
Other import-cargo tests refer to the same URL.
They are almost the same tests, so this should have little impact. If there are any changes, I would like to make a separate PR.

Reply:
I see.